tools/integration_tests/run_tests_mounted_directory.sh

```diff
@@ -133,7 +133,11 @@ gcsfuse --implicit-dirs $TEST_BUCKET_NAME $MOUNT_DIR
 GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/list_large_dir/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR --testbucket=$TEST_BUCKET_NAME
 sudo umount $MOUNT_DIR
 
-# Run integration tests for list_large_dir with --only-dir mounting.
```

Comment:
Removed this as we are not running in our [code](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/list_large_dir/list_large_dir_test.go#L50).